.gitignore

```diff
@@ -2,3 +2,4 @@
 .DS_Store
 libs/protos/build/ts/node_modules
 settings.json
+libs/protos/build/ruby/*.gem
```

Comment:
Should we not leave the .gem in the build dir? We do it with other langs - any reason why not to?